### PR TITLE
Move beat specific logic to docker-compose

### PR DESCRIPTION
**Pull request by ruflin**
This will make it possible to better reuse the base docker image.

### CODE REVIEW

File: libbeat/docker-compose.yml [hunk 1]
```diff
@@ -12,8 +12,13 @@ libbeat:
     - LS_TLS_PORT=5055
     - ES_USER=beats
     - ES_PASS=testing
+    # Setup work environment
+    - LIBBEAT_PATH /go/src/github.com/elastic/beats/libbeat
```
Comment (urso) [inline libbeat/docker-compose.yml]: does this work without `=`?

Comment (ruflin) [inline libbeat/docker-compose.yml]: I don't know. I will add the =

File: libbeat/docker-compose.yml [hunk 2]
```diff
@@ -12,8 +12,13 @@ libbeat:
...
   env_file:
     - build/test.env
+  working_dir: /go/src/github.com/elastic/beats/libbeat
```
Comment (ruflin) [inline libbeat/docker-compose.yml]: I tested using LIBBEAT_PATH variable here, doesn't work

File: libbeat/docker-compose.yml (hunk 1 above)
Comment (ruflin) [inline libbeat/docker-compose.yml]: fixed

**Comment by urso** [comment]
This re-enables integration tests using shield.

**Pull request merged**